android/app/build.gradle

```diff
@@ -149,8 +149,8 @@ android {
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
         multiDexEnabled rootProject.ext.multiDexEnabled
-        versionCode 1001012419
-        versionName "1.1.24-19"
+        versionCode 1001012422
+        versionName "1.1.24-22"
```

Comment:
How are these changes related to this PR?

Reply:
I am not sure. But when we merge main, do the new changes show in PR changes?

Reply:
I think they shoudn't show, I'm not understanding why its showing, we are currently using version 1.1.24-22 it shouldn't show as a change

Reply:
I'll try to revert the last commit changes